source/change-streams/tests/unified/change-streams-clusterTime.yml

```diff
@@ -0,0 +1,40 @@
+description: "change-streams-clusterTime"
+schemaVersion: "1.3"
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+
+runOnRequirements:
+  - minServerVersion: "4.0.0"
```

Comment:
Why 4.0.0?

Reply:
The field was new as of 4.0 (see the description of DRIVERS-471).